ghedesigner/ghe/gfunction.py

```diff
@@ -32,13 +32,7 @@ def calculate_g_function(
     pipe,
     grout,
     soil,
-    n_segments=8,
-    end_length_ratio=0.02,
-    segments="unequal",
-    solver="equivalent",
-    boundary="MIFT",
-    segment_ratios=None,
-    disp=False,
+    boundary_condition="MIFT",
```

Comment:
Maybe this got lost along the way, but most of these were never exposed to users. I've hardcoded them for now, and we can figure out the right way to expose them, if needed, in the future.